src/SessionSetup/ConnectToStadiaWidget.cpp

```diff
@@ -92,25 +101,69 @@ ConnectToStadiaWidget::ConnectToStadiaWidget(QWidget* parent)
                    &ConnectToStadiaWidget::UpdateRememberInstance);
   QObject::connect(ui_->refreshButton, &QPushButton::clicked, this,
                    [this]() { emit InstanceReloadRequested(); });
+  QObject::connect(ui_->instancesFilterLineEdit, &QLineEdit::textChanged, &instance_proxy_model_,
+                   &QSortFilterProxyModel::setFilterFixedString);
+  QObject::connect(ui_->refreshButton_2, &QPushButton::clicked, this,
+                   [this]() { emit InstanceReloadRequested(); });
```

Comment:
yes exactly. Once the flag is removed, `refreshButton` will get removed and `refreshButton_2` will be renamed to `refreshButton`. Because the position of the refresh button is slightly different in the new UI and because its currently  hidden behind a flag, both buttons are necessary. Only one of them will ever be visible, depending on the flag being set or not